dist/es6/themeOverride.js

```diff
@@ -279,44 +291,107 @@ export var themeOverride = {
     },
     icons: {
       include: {
-        icon: Success,
+        icon: TickCircle,
         extend: {
-          color: 'status-ok',
+          color: 'accent-1',
           size: 'large'
         }
       },
       exclude: {
-        icon: Block,
+        icon: TickCircle,
         extend: {
-          color: 'status-error',
+          color: 'error',
           size: 'large'
         }
       }
     }
   },
+  pagination: {
+    border: {
+      color: 'none'
+    },
+    button: {
+      active: {
+        fontWeight: 'bold',
+        background: 'light-3',
+        border: {
+          color: 'light-3'
+        },
+        color: 'inherit'
+      }
+    }
+  },
+  radioButton: {
+    border: {
+      width: '5px'
+    },
+    extend: {
+      marginRight: '8px'
+    },
+    container: {
+      extend: function extend(_ref) {
+        var checked = _ref.checked,
+            theme = _ref.theme;
+        return {
+          fontWeight: checked && 600,
+          color: normalizeColor('dark-3', theme)
+        };
+      }
+    }
+  },
+  rangeInput: {
+    track: {
+      color: 'dark-3',
+      height: '4px'
+    },
+    extend: function extend(_ref2) {
+      var disabled = _ref2.disabled;
+      return disabled && 'opacity: .3; cursor: not-allowed !important;';
+    },
+    thumb: {
+      extend: function extend(_ref3) {
+        var disabled = _ref3.disabled;
+        return disabled && 'cursor: not-allowed !important;';
+      }
+    }
+  },
+  tabs: {
+    // background: undefined,
+    // extend: undefined,
+    gap: 'large',
+    header: {
+      // background: undefined,
+      extend: {
+        'padding-left': baseSpacing * 1.2 + "px",
+        'border-bottom': '1px solid #E8E7E7 '
+      }
+    }
+  },
   textInput: {
     container: {
       extend: function extend() {
         return "\n          height: 100%;\n        ";
       }
     },
-    extend: function extend(_ref) {
-      var plain = _ref.plain,
-          focus = _ref.focus,
-          reverse = _ref.reverse,
-          icon = _ref.icon,
-          theme = _ref.theme,
-          readOnly = _ref.readOnly;
+    extend: function extend(_ref4) {
+      var plain = _ref4.plain,
+          focus = _ref4.focus,
+          reverse = _ref4.reverse,
+          icon = _ref4.icon,
+          theme = _ref4.theme,
+          readOnly = _ref4.readOnly,
+          error = _ref4.error;
       return _extends({
         paddingTop: baseSpacing / 1.78 + "px",
         paddingBottom: baseSpacing / 1.78 + "px",
         boxShadow: 'none',
         height: '100%',
+        fontSize: '12px',
```

Comment:
this will not allow dynamic font size based on screen size, if we have to implement that in future

Reply:
For now, instead of hard coding reading fontSize from global.font.size of theme object